Logging.sln

```diff
@@ -76,32 +76,50 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "NLogTarget.Tests.Shared", "
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "TraceListener.Tests.Shared", "src\Adapters.Tests\TraceListener.Tests.Shared\TraceListener.Tests.Shared.shproj", "{17EE52DA-5460-4BD2-A70F-529D2A94C938}"
 EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "CommonShared", "src\Adapters\CommonShared\CommonShared.shproj", "{587B624B-8C64-498E-93D7-A2D2ABC17EAB}"
```

Comment:
I did not do that manually, that is what VS is adding and I would not want to change it since it is working as expected.

---
In reply to: [66372033](https://github.com/Microsoft/ApplicationInsights-dotnet-logging/pull/34#discussion_r66372033) [](ancestors = 66372033)